wn/_core.py

```diff
@@ -902,7 +902,7 @@ def synset(self) -> Synset:
             Synset('pwn-03325088-n')
 
         """
-        return synset(id=self._synset_id)
```

Comment:
This is cleaner than the solution I came up with. Maybe I'll adapt it for the other secondary query methods :)

Reply:
Glad you like it.
BTW: Woohoo for my first ever accepted pull request! It's official, I am now an open source library contributor!   :-)

Reply:
Thanks for your contribution! :beer: